src/ducks/Committees.js

```diff
@@ -139,10 +140,8 @@ export const editCommittee = (title, description, chair, oldTitle) => {
 						chair: chair,
 						level: level
 					}))
-					.then(() => alert("Committee Edited!", "Successful"))
-					.catch(() => {
-						alert("Committee could not be Edited!", "Failure");
-					});
+					.then(() => Alert.alert("Committee Edited!", { type: "success", title: "Successful" }))
+					.catch(() => Alert.alert("Committee could not be Edited!", { type: "error", title: "Failure" }));
```

Comment:
I know, you didn't originally write these messages but maybe we should just change the action words like Edited to lowercase in all of these messages for consistency